composer.json

```diff
@@ -18,10 +18,7 @@
         "symbiote/silverstripe-queuedjobs": "if using gridfieldqueuedexport you can batch background jobs with this module"
     },
     "extra": {
-        "project-files": [],
-        "branch-alias": {
-            "dev-master": "1.x-dev"
-        }
```

Comment:
Did you mean to do this? The master branch is where branch aliases are most useful

Reply:
I guess we don't need the `project-files` either

Reasoning was in my project composer.json I have
```
"silverstripe/recipe-form-building": "1.x-dev"
```
however it annoyingly installs
```
            "name": "silverstripe/recipe-form-building",
            "version": "dev-master",
```
dev-master is old, and includes `"silverstripe/userforms": "5.4.x-dev"` instead of `"silverstripe/userforms": "5.x-dev"`

Reply:
Right, sounds like the fix is to change master's alias to 2.x-dev then - it must've been forgotten about when doing a merge up from the 1 branch when it was created

Reply:
ok, can do, though first could you help me understand what the use-case for using and maintaining these branch-alias is?  I'm not sure what problem they are meant to solve?

Reply:
Docs: https://getcomposer.org/doc/articles/aliases.md

The main benefit is that people can install a 2.x release line (of this module in this example) before it has a branch or tag, then when we switch master to 2 and stabilise they don't have to change constraints. This is especially important when dealing with dependent packages, where you might not require it explicitly, but one of your packages does (and can do so with a branch alias)